x-pack/plugins/security_solution/cypress/e2e/detection_rules/bulk_edit_rules.cy.ts

```diff
@@ -132,6 +132,7 @@ describe('Detection rules, bulk edit', () => {
     login();
   });
   beforeEach(() => {
+    clearSessionStorage();
```

Comment:
@MadameSheema this looks as a simple and proper fix since a rules table state is persisted in the session storage and interferes to the bulk edit e2e tests.